MAINTAINERS.md

```diff
@@ -2,40 +2,14 @@
 
 ## Maintainer Scopes, GitHub Roles and GitHub Teams
 
```

Comment:
I'm not exactly sure what to do with this file. Tried to copy what was done in aca-py, but still some hyperledger references.

Reply:
I can do a follow up PR to address it.  I think I know what is needed.